src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,5 +9,6 @@
     public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
     public static final Prefix PREFIX_EXPENSE_CATEGORY = new Prefix("c/");
     public static final Prefix PREFIX_AMOUNT = new Prefix("a/");
+    public static final Prefix PREFIX_EXPENSE_DATE = new Prefix("D/");
```

Comment:
i think we should keep the lower case consistent and not have two d commands... maybe t/ would be a possible substitute

Reply:
Shall change